Documentation/CraftingNuGetPackages.md

```diff
@@ -0,0 +1,69 @@
+# Crafting a NuGet Package for MSB4U
+
+> MSBuildForUnity consumes .NET based packages without any modifications to the package. This documentation describes how C++, WinRT or packages with Unity-specific contents can easily be adapted to work with MSBuildForUnity.
+
+## Quickstart Summary
+
+Through the use of a [Build Targets or Props](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#include-msbuild-props-and-targets-in-a-package) file inside a NuGet package, we can leverage properties provided by MSBuildForUnity to appropriately select the contents to bring into the Unity project. The following snippet is taken from [Microsoft.Windows.MixedReality.DotNetWinRT](https://www.nuget.org/packages/Microsoft.Windows.MixedReality.DotNetWinRT/) NuGet package. It brings in the `Unity` folder packaged in the root of the NuGet package to the output folder inside Unity.
```

Comment:
Per Chris's suggestion down below, it does feel like there's some opportunity to combine the list of steps along with this quickstart summary - i.e. it feels like this section right here somewhat describes the final outcome/state that should be accomplished (i.e. the specific props file that you need to have in order to make it work).

It also feels like this particular thing is based on step 5 of the list below - it's really useful having a concrete example of what needs to be done in step 5 - so this type of content is great.

I think maybe a suggestion here is it could show up as a subsection of step 5 - i.e. here's step 5, and here's an example of how to accomplish it.